uint64 timestamp	# Time since system start (microseconds)
uint64 error_count	# Timer overcapture error flag (AUX5 or MAIN5)
uint32 pulse_width	# Pulse width, timer counts
uint32 period		# Period, timer counts
